From: Sergio Durigan Junior <sergiodj@debian.org>
Date: Mon, 11 May 2020 18:51:39 -0400
Subject: Use RuntimeDirectory to create /run/squid

Instead of installing the /run/squid directory, which goes against
Debian Policy, we instruct systemd to automatically create it for us
when the service is started.
---
 tools/systemd/squid.service | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/tools/systemd/squid.service b/tools/systemd/squid.service
index 4094f0c..b1b0526 100644
--- a/tools/systemd/squid.service
+++ b/tools/systemd/squid.service
@@ -12,7 +12,10 @@ After=network.target network-online.target nss-lookup.target
 
 [Service]
 Type=notify
-PIDFile=/var/run/squid.pid
+PIDFile=/run/squid.pid
+Group=proxy
+RuntimeDirectory=squid
+RuntimeDirectoryMode=0775
 ExecStartPre=/usr/sbin/squid --foreground -z
 ExecStart=/usr/sbin/squid --foreground -sYC
 ExecReload=/bin/kill -HUP $MAINPID
